Extensions/ExternalTfs/Src/Tasks/DownloadExternalBuildArtifacts/task.loc.json

```diff
@@ -10,25 +9,40 @@
   "version": {
     "Major": 15,
     "Minor": 264,
-    "Patch": 2
+    "Patch": 3
```

Comment:
I am not a bit expect in our task versioning, but IMHO adding new Endpoint typo support is not patch, minor at least.

Reply:
Makes sense. I agree.